kubeflow/trainer/rhai/transformers.py

```
@@ -323,9 +324,83 @@ def checkpoint_in_progress(self):
+                    print(f"[Kubeflow] Downloading checkpoint: {name}", flush=True)
+                    self.fsspec.get(
+                        f"{self.cloud_storage_base_path}/{name}", args.output_dir, recursive=True
+                    )
+                    break
+
+            # Barrier to wait for rank 0 checkpoint download to complete
+            try:
+                if dist.is_initialized():
+                    dist.barrier()
+            except Exception:
+                pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)